### PR TITLE
Remove unnecessary transaction verification

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

Transactions can never fail verification during bootstrapping anymore. Because they are already verified to have been included in the network prior to the X-chain linearization.

## How this works

Deletes redundant pre-linearization tx verification.

## How this was tested

CI.

### CODE REVIEW

File: vms/avm/vm.go [hunk 1]
```diff
@@ -460,17 +460,21 @@ func (vm *VM) ParseTx(_ context.Context, bytes []byte) (snowstorm.Tx, error) {
 		return nil, err
 	}
 
+	err = rawTx.Unsigned.Visit(&txexecutor.SyntacticVerifier{
```
Comment (patrick-ogrady) [inline vms/avm/vm.go:463]: Why not use `UniqueTx` inside of this?

Comment (StephenButtolph) [reply vms/avm/vm.go:463]: `UniqueTx` is being removed (also `SyntacticVerifier` is not taking in an interface, we can't use `UniqueTx` for that)

**Review by coffeeavax — APPROVED** [review]

**Comment by StephenButtolph** [comment]
While this verification isn't needed (w.r.t. the security assumptions of the network) it can still detect DB corruption... So we'll keep it (for now).

**Comment by StephenButtolph** [comment]
Closed for #1654

**Pull request closed without merging**